# Axon information message from i-Building.
uint32  axon_msg		# AXON message ID
float32 left_mspeed		# Left motor (unit: m/s)
float32 right_mspeed		# Right motor (unit: m/s)
uint32 	ad_start_flag		# auto docking start command flag
uint32  ec_get_flag		# get motor encoder counts flag
uint32  ms_set_respone		# motor speed setting respone flag
uint32 	ad_set_respone		# auto docking start respone
int32 	left_motor_ec		# Left motor encoder counts value
int32 	right_motor_ec		# Right motor encoder counts value

